# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../strace/glibc-2.5.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

GlibC-2.5 defines it but it does not build. Disable it for now.

  - Rene Rebe <rene@exactcode.de>

--- strace-4.5.14/strace.c.vanilla	2006-10-20 13:38:24.000000000 +0200
+++ strace-4.5.14/strace.c	2006-10-20 13:40:51.000000000 +0200
@@ -2251,6 +2251,7 @@
 			if (!cflag
 			    && (qual_flags[WSTOPSIG(status)] & QUAL_SIGNAL)) {
 				unsigned long addr = 0, pc = 0;
+#undef PT_GETSIGINFO /* does not build */
 #ifdef PT_GETSIGINFO
 #				define PSR_RI	41
 				struct siginfo si;
